packages/manager/modules/pci/package.json

```diff
@@ -45,7 +45,6 @@
     "@ovh-ux/ng-ovh-payment-method": "^2.1.0",
     "@ovh-ux/ng-ovh-proxy-request": "^1.0.0-beta.0",
     "@ovh-ux/ng-ovh-responsive-popover": "^5.0.0-beta.0",
-    "@ovh-ux/ng-ovh-sidebar-menu": "git+https://github.com/ovh-ux/ng-ovh-sidebar-menu.git#feat/always-open",
```

Comment:
Related to this PR ?

Reply:
yes as the refactor is needed to add the tracking this was handle in the same PR. I can split this to have as many PRS as commits if it seems better